dynawo/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@
 # This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
 
 # Minimum required (for ExternalProject_Add)
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.9.6 FATAL_ERROR)
```

Comment:
we need to make sure this compiles on the production environment (currently using cMake 3.6.2)

Reply:
It does. We decided a while back it was the new version, the last one to compile with c++98.